packages/react-core/src/components/Dropdown/DropdownWithContext.tsx

```diff
@@ -96,6 +98,9 @@ export class DropdownWithContext extends React.Component<DropdownProps & OUIAPro
           const BaseComponent = baseComponent as any;
           const menuContainer = (
             <DropdownMenu
+              // This removes the `position: absolute` styling from the `.pf-c-dropdown__menu`
+              // allowing the menu to flip correctly
+              {...(isFlipEnabled && { style: { position: 'revert', minWidth: 'min-content' } })}
```

Comment:
Looks like this `min-width` is needed because the positioned popper element is assigned a `width`, which is the same width as the toggle (as far as I can tell). So the `__menu` is currently overflowing the positioned popper element, even though it looks OK.

On the context selector, however, here is what a non-positioned versus popper positioned menu looks like since the positioned menu has a fixed width:

non-positioned
<img width="704" alt="Screen Shot 2022-05-26 at 2 58 15 PM" src="https://user-images.githubusercontent.com/35148959/170568496-fb49d218-f845-40e9-b181-dec1f3550b3f.png">

popper positioned
<img width="472" alt="Screen Shot 2022-05-26 at 2 58 28 PM" src="https://user-images.githubusercontent.com/35148959/170568458-2f45e196-4f45-487d-b488-012d643d3d34.png">

Can we nix the width? Or maybe assign a `min-width` instead?